android/src/main/java/com/reactnativestripesdk/customersheet/CustomerSheetFragment.kt

```diff
@@ -89,7 +91,11 @@ class CustomerSheetFragment : Fragment() {
       .googlePayEnabled(googlePayEnabled)
       .headerTextForSelectionScreen(headerTextForSelectionScreen)
       .preferredNetworks(mapToPreferredNetworks(arguments?.getIntegerArrayList("preferredNetworks")))
+      .allowsRemovalOfLastSavedPaymentMethod(allowsRemovalOfLastSavedPaymentMethod ?: true)
```

Comment:
Isn't this always defaulting to true? if I inject `allowsRemovalOfLastSavedPaymentMethod` being `false` this code would default to `true` @charliecruzan-stripe

Reply:
no, the elvis operator tests whether the left hand side is non-null: https://kotlinlang.org/docs/null-safety.html#elvis-operator

here's some sample code to prove this will evaluate to `false` if `allowsRemovalOfLastSavedPaymentMethod` is false